extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/CodeAuthenticationMechanism.java

```diff
@@ -294,6 +320,7 @@ public void accept(SecurityIdentity identity) {
                                                 finalRedirectUri += ("?" + finalUserQuery);
                                             }
                                             LOG.debugf("Final redirect URI: %s", finalRedirectUri);
+                                            //TODO: Does this need to check autoRedirect
```

Comment:
You can remove this line, it is the same origin redirect to drop the parameters returned back from OIDC (`code`, etc)

Reply:
Yes. I meant to ask about this so I could remove it. Will do now.